### DISCUSSION TITLE
not able to pair - blank timezone window when pairing

### DISCUSSION THREAD

**Discussion by GeroBW** [hass-emulated-hue/core#402] [General]
Hello, I am not certain if this issue is related to the hue app or emulated hue. the pairing process currently gets stuck on the "select timezone" step. When trying to select the timezone, I get a blank (browser?) window.

**Comment by allgrinder** [comment]
I have the same issue

**Comment by metzgek** [comment]
Same issue here.  I’m able to add my current Hue hub through the app, but no luck with the emulated hub through HA.  It may still be a Hue app issue, because my current hub already has a time zone assigned.  Maybe it just doesn’t ask for it again if it’s already assigned?

**Comment by alexyao2015** [comment]
The official hue app does not currently work with this project due to the lack of the v2 api. See #257 for more information.
